src/color.rs

```diff
@@ -1116,6 +1116,19 @@ impl Color {
 
         (hue.round(), saturation, lightness)
     }
+
+    /// Returns rgba string in CSS functional syntax
+    ///
+    /// ```rust
+    /// use turtle::Color;
+    /// let color = Color::rgba(255., 125., 125., 0.3);
+    /// let rgba = color.rgba_string();
+    /// assert_eq!(rgba, "rgba(255, 125, 125, 0.3)");
+    /// ```
+
+    pub fn rgba_string(&self) -> String {
```

Comment:
I appreciate the thought to make this a method, but I'm not sure if it would be best to have this as part of the public API. Could you make this `pub(crate)` so the method isn't usable publically? Even better would be to put it as an inner function in `renderer::save_svg` since it's only being used in that one place.